public/scripts/ssooverride.js

```diff
@@ -83,33 +89,53 @@
         }, false);
       });
 
-      var thimbleAuth = new WebmakerLogin({
-        csrfToken: csrf,
-        showCTA: false
-      });
+      var basicQuery = "";
+
+      basicQuery += "?client_id=" + oauthClientId;
+      basicQuery += "&response_type=code";
+      basicQuery += "&scopes=user email";
 
+      // Signup login flow
       joinEl.on('click', function() {
-        thimbleAuth.create();
+        var state = uuid.v4();
+        var oauthRoute = "/login/oauth/authorize";
+
+        var query = authHostname + oauthRoute + basicQuery;
+        query += "&state=" + state;
+        query += "&action=signup";
+
+        setStateCookie(state);
+
+        window.location = query;
       });
+      // Login flow
       loginEl.on('click', function() {
-        thimbleAuth.login();
+        var state = uuid.v4();
+        var oauthRoute = "/login/oauth/authorize";
+
+        var query = authHostname + oauthRoute + basicQuery;
```

Comment:
As commented above, this can be condensed to one line, all the += is unnecessary.